tests/bug-1101.sil

```diff
@@ -0,0 +1,7 @@
+\begin[class=jplain,layout=tate,papersize=a4]{document}
+\script[src=packages/tate]
+\neverindent\nofolios
+\font[family=Noto Sans CJK JP,language=ja]
+
+\latin-in-tate{SILE}と\latin-in-tate{Signal}。
```

Comment:
Running this test for me with the `tate.lua` fix reverted looks fine. I don't think this test is adequate to trigger the bug that #1102 represents, but I'm not sure what is either.

Reply:
Per Fred's comments on the issue and my testing, the word "Signal" does not trigger this bug whereas "foobar" or "Latin" in the same context do. I've updated the test with one of those so that running the test without the fix in place actually fails as expected.